src-django/api/tests/test_procedures.py

```diff
@@ -9,28 +9,48 @@
 
 
 class ProcedureTest(TestCase):
+
     def setUp(self):
         self.client = Client()
         self.token = Token.objects.get(user=factories.UserFactory())
-        self.procedure_url = '/api/procedures/'
+        self.procedure_url = '/api/procedures'
         self.data = {
             "title": "Example Title",
             "author": "An Author"
         }
+        self.user = factories.UserFactory()
+        initialize_permissions
 
-    @initialize_permissions
     def test_created_procedure_has_correct_owner(self):
-        user = factories.UserFactory()
-
         response = self.client.post(
             path=self.procedure_url,
             data=json.dumps(self.data),
             content_type='application/json',
-            HTTP_AUTHORIZATION=add_token_to_header(user, self.token)
+            HTTP_AUTHORIZATION=add_token_to_header(self, self.user, self.token)
```

Comment:
When I removed the decorator from the method and added it to the `setUp` method, the call to `add_token_to_header` was throwing an error saying it needed a 3rd argument, where it's required params were `(self, user, token)`.